src/main/java/com/parasoft/demoapp/service/GlobalPreferencesService.java

```diff
@@ -216,6 +229,29 @@ private void refreshProxyDataSource(GlobalPreferencesEntity currentPreferences){
         parasoftJDBCProxyService.refreshParasoftJDBCProxyDataSource();
     }
 
+    private void refreshDestination(GlobalPreferencesEntity currentPreferences) throws GlobalPreferencesNotFoundException, GlobalPreferencesMoreThanOneException {
```

Comment:
Remove the GlobalPreferencesNotFoundException, GlobalPreferencesMoreThanOneException since there is not exception to throw.